apps/docs/components/blocks/RatingForms.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: DefaultLayout
+hideBreadcrumbs: true
+description: The e-commerce rating and review solution provides a platform for users to rate their e-commerce experiences. Users can assign ratings based on their satisfaction, product quality, delivery experience, and other relevant factors. These ratings offer valuable quantitative information that helps other potential buyers gauge the overall reputation and quality of products and services offered on the platform.
+hideToc: true
+---
+# Rating Forms
+
+{{ $frontmatter.description }}
+
+## Product Rating
+
+The Product Rating block serves as a basic yet effective solution for implementing a 1-5 stars rating system commonly used in eCommerce platforms. It enables users to easily express their opinion about a particular item or experience.
+
+It allows for customization through the use of RatingButton basic component. This feature permits users to provide more specific ratings, enhancing the granularity of their feedback.
+
+For additional customizations and more advanced functionality, you can consult our documentation on the Modal page.
```

Comment:
It'd be nice to link the Modal page here. I'm not sure if there's a better way that `<!-- vue --> <!-- react -->` blocks here.

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2849/commits/d3797690dbd108bf5bf4029e2c4270327ee10e2a